systems/trajectory_optimization/hybrid_dircon.cc

```diff
@@ -24,34 +22,39 @@ using drake::systems::trajectory_optimization::MultipleShooting;
 using drake::multibody::MultibodyPlant;
 using drake::AutoDiffXd;
 using drake::VectorX;
-using drake::solvers::VectorXDecisionVariable;
 using drake::symbolic::Expression;
 using Eigen::VectorXd;
 using Eigen::MatrixXd;
 using std::vector;
 
+// HybridDircon constructor
 template <typename T>
-HybridDircon<T>::HybridDircon(
-    const MultibodyPlant<T>& plant,
-    vector<int> num_time_samples,
-    vector<double> minimum_timestep,
-    vector<double> maximum_timestep,
-    vector<DirconKinematicDataSet<T>*> constraints,
-    vector<DirconOptions> options) :
-        MultipleShooting(plant.num_actuators(),
-          plant.num_positions() + plant.num_velocities(),
-          std::accumulate(num_time_samples.begin(),
-          num_time_samples.end(), 0) - num_time_samples.size() + 1, 1e-8, 1e8),
-    plant_(plant),
-    constraints_(constraints),
-    num_modes_(num_time_samples.size()),
-    mode_lengths_(num_time_samples),
-    v_post_impact_vars_(NewContinuousVariables(plant.num_velocities() *
-                        (num_time_samples.size() - 1), "v_p")) {
+HybridDircon<T>::HybridDircon(const MultibodyPlant<T>& plant,
+                              vector<int> num_time_samples,
+                              vector<double> minimum_timestep,
+                              vector<double> maximum_timestep,
+                              vector<DirconKinematicDataSet<T>*> constraints,
+                              vector<DirconOptions> options)
+    : MultipleShooting(plant.num_actuators(),
+                       plant.num_positions() + plant.num_velocities(),
+                       std::accumulate(num_time_samples.begin(),
+                                       num_time_samples.end(),
+                                       0) - num_time_samples.size() + 1,
+                       1e-8,
+                       1e8),
+      plant_(plant),
+      constraints_(constraints),
+      num_modes_(num_time_samples.size()),
+      mode_lengths_(num_time_samples),
+      v_post_impact_vars_(NewContinuousVariables(
+          plant.num_velocities() *
```

Comment:
Also here

Reply:
Same comment. I couldn't tell how it is off. I think this is the same as drake's style?